lumicks/pylake/kymotracker/kymotracker.py

```diff
@@ -481,6 +506,7 @@ def refine_tracks_centroid(tracks, track_width=None, bias_correction=True):
             interpolated_tracks[j]._kymo,
             interpolated_tracks[j]._channel,
             half_width_pixels,
+            interpolated_tracks[j]._minimum_observable_duration,
```

Comment:
not introduced here but could do a quick fixup replacing `interpolate_tracks[j]` with `track` as it's not actually used

Reply:
Good catch!